packages/db/src/database.ts

```diff
@@ -6,12 +6,6 @@ import { tableNames } from './tables';
 
 export const isRecordDefined = <T>(record: T) => record !== undefined;
 
-export const doesRecordExist = async<T>(
-  params: Partial<T>,
-  knex: Knex,
-  tableName: string
-): Promise<boolean> => isRecordDefined(await knex<T>(tableName).where(params).first());
-
```

Comment:
Should you remove `getRecordCumulusId` as well?

Reply:
Yup, good catch!

Reply:
46572d3010a007041ed4b38d9098800f68513566